### PR TITLE
README: mention Command Line Interface Pages project

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

## Notes

[Render](https://github.com/emilyseville7cfg-better-tldr/prototypes/tree/main/btldr-view) and tool prototypes are [ready to use](https://github.com/emilyseville7cfg-better-tldr/prototypes).

image
image

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
@kbdharun, @navarroaxel, please review this PR.

File: README.md [hunk 1]
```diff
@@ -108,6 +108,11 @@ to see the overall progress of all translations, and which translations are miss
 
 ## Similar projects
 
+- [Better TlDr](https://github.com/emilyseville7cfg-better-tldr)
+  allows you to write almost the same pages as TlDr but in a standardized way
```
Comment (SethFalco) [inline README.md:112]: > almost the same pages as TlDr

I wouldn't call it "almost the same", just "similar".

File: README.md [hunk 2]
```diff
@@ -108,6 +108,11 @@ to see the overall progress of all translations, and which translations are miss
...
+  and with rich IDE-like facilities like syntax highlighting, refactorings and
+  linting. It is designed to be a replacement for TlDr project.
```
Comment (SethFalco) [inline README.md:114]: > It is designed to be a replacement for TlDr project.

I would refrain from calling it a replacement. An "alternative" could be fine, or even better saying that it's inspired by or derived from this project, but calling it a replacement makes tldr-pages sound deprecated which is not the case.

**Review by SethFalco — COMMENTED** [review]
Overall, I think this is interesting but as the project is literally brand new, I think it'd be best to wait at least a month or so and ensure it's still being maintained before it's added.

Typically, I'd expect the project to meet some kind of threshold of notability and/or support before it's listed here. Though other maintainers can pitch in what they think.

**Comment by SethFalco** [comment]
I'd also be a bit wary about calling it "better-tldr" but that's up to other maintainers.

While tldr is not trademarked, keep in mind that open-source licenses are only a license of copyright, not trademarks. If tldr were to grow to be a larger project or especially commercially supported, the name may be considered infringing as it piggybacks off the notability of this project.

Setting trademarks aside, it still may give a sense of affiliation. I'd like other maintainers to give their thoughts here.

File: README.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply README.md]: I've removed the sentence about replacement even it's the true goal of my project. I believe that my project is that TlDr should be instead it is now.

Comment (SethFalco) [reply README.md]: What you believe you can put in your README. But in this README it needs to be an _objective_ description of your project, not your goals or opinions on it!

Comment (EmilySeville7cfg) [reply README.md]: Noted.

**Comment by EmilySeville7cfg** [comment]
> Typically, I'd expect the project to meet some kind of threshold of notability and/or support before it's listed here. Though other maintainers can pitch in what they think.

Sure, that's what I need. I promoting my project now, but I feel that mentioning it in TlDr or similar projects would be useful to gain more popularity.

**Comment by SethFalco** [comment]
> It can be renamed I guess. For smth like clip which can be stand for [c]ommand [l]ine [i]nterface [p]ages. I thought now, and decided that it's not a big issue to rename project at this stage.

We'll see what other maintainers say first, but just want to say regardless of other opinions, I think clip (and the expanded version) is a solid name.

**Comment by kbdharun** [comment]
> > It can be renamed I guess. For smth like clip which can be stand for [c]ommand [l]ine [i]nterface [p]ages. I thought now, and decided that it's not a big issue to rename project at this stage.
> 
> We'll see what other maintainers say first, but just want to say regardless of other opinions, I think clip (and the expanded version) is a solid name.

Agreed, it indeed is an excellent name for it.

**Comment by kbdharun** [comment]
While we are at this should we remove the editor plugin lines from README?

**Comment by EmilySeville7cfg** [comment]
> While we are at this should we remove the editor plugin lines from README?

Yes, I am not interested in supporting it anymore.

**Comment by EmilySeville7cfg** [comment]
Project has been renamed.

**Comment by EmilySeville7cfg** [comment]
@SethFalco, project is still maintained. Month is already passed. :)

**Comment by SethFalco** [comment]
At this stage I guess it gets a pass for me. Will need to amend your PR to include the new name/description, though!

What does everyone else think?

I still feel in general there should be a threshold of notability before being listed, but perhaps this should be defined separately for future?

**Comment by EmilySeville7cfg** [comment]
> Will need to amend your PR to include the new name/description, though!

Just fixed. 😄

File: README.md [hunk 3]
```diff
@@ -108,10 +108,8 @@ to see the overall progress of all translations, and which translations are miss
 
 ## Similar projects
 
-- [Better TlDr](https://github.com/emilyseville7cfg-better-tldr)
-  allows you to write similar pages as TlDr but in a standardized way
-  and with rich IDE-like facilities like syntax highlighting, refactorings and
-  linting.
+- [Command Line Interface Pages](https://github.com/command-line-interface-pages)
+  allows you to write standardized help pages for command CLI, directories and configs.
```
[the remark this answers is not in the document]
Comment (EmilySeville7cfg) [reply README.md]: Fixed.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@SethFalco Any updates on this, Can it be merged?

**Comment by EmilySeville7cfg** [comment]
I am so happy it's merged so I am ready hug everyone participated in this PR. Just come here. 🤗

**Pull request merged**